### ISSUE TITLE
WIN10 Error

### ISSUE THREAD

**Issue by Jylpah** [achillesrasquinha/pipupgrade#28]
Hi, I just installed pipupgrade and tried running it on Win10 / Python 3.7.3. I have one dependency package that is too new to satisfy the requirements for another package. I did not start to debug this. 

```Successfully installed pipupgrade-1.5.0
PS C:\Uusers\USER\> pipupgrade --latest --yes
c:\users\USER\appdata\local\programs\python\python37\lib\site-packages\requests\__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.2) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
[0;93mChecking...[0m
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python37\Scripts\pipupgrade-script.py", line 11, in <module>
    load_entry_point('pipupgrade==1.5.0', 'console_scripts', 'pipupgrade')()
  File "c:\users\USER\appdata\local\programs\python\python37\lib\site-packages\pipupgrade\cli\__init__.py", line 14, in wrapper
    return fn(**params)
  File "c:\users\USER\appdata\local\programs\python\python37\lib\site-packages\pipupgrade\commands\__init__.py", line 162, in command
    format = "json", pip_exec = pip_)
  File "c:\users\USER\appdata\local\programs\python\python37\lib\site-packages\pipupgrade\_pip.py", line 65, in call
    return popen(*params, output = True)
  File "c:\users\USER\appdata\local\programs\python\python37\lib\site-packages\pipupgrade\util\system.py", line 69, in popen
    raise sp.CalledProcessError(code, command)
subprocess.CalledProcessError: Command 'None list --outdated --format json' returned non-zero exit status 1.
PS C:\Uusers\USER\>```

**Comment by achillesrasquinha** [comment]
Hi @Jylpah, I'll have a fix dispatched for this soon. There's currently a workaround for this mentioned at #27

**Comment by achillesrasquinha** [comment]
Hi @Jylpah, I'll have a fix dispatched for this soon. There's currently a workaround for this mentioned at #27

**Comment by achillesrasquinha** [comment]
@Jylpah This has been fixed, pending for release. Please update using `pipupgrade --self` and try again.

**Comment by achillesrasquinha** [comment]
Released with 1.5.1

**Comment by achillesrasquinha** [comment]
Closing this. Kindly reopen if persists.
